### PR TITLE
show loading indicator when execute query is in progress

### PR DESCRIPTION
**Pull request by m-kharbat**
buf6ixbzqc

### CODE REVIEW

File: app/views/console.html [hunk 1]
```diff
@@ -42,7 +42,10 @@
 
             <div class="cr-console-header__options">
               <div class="cr-console-header__options__item--hint">{{ 'CONSOLE.HINT' | translate }}</div>
-              <div id="execute-btn" class="cr-console-header__options__item--btn cr-button cr-button--primary" ng-click="execute()" type="submit" class="ladda-button" data-style="expand-left" translate>{{ 'CONSOLE.EXCUTE_QUERY' }}</div>
+              <div id="execute-btn" class="cr-console-header__options__item--btn cr-button cr-button--console" ng-click="execute()" type="submit" >
+                <div ng-class="loading ? 'hidden' : ''" translate>{{ 'CONSOLE.EXCUTE_QUERY' }}</div>
```
Comment (chaudum) [inline app/views/console.html]: could you replace `ng-class="loading ? 'hidden' : ''"` with `ng-if="!loading"`?

Comment (chaudum) [inline app/views/console.html]: or is it for keeping the width?

Comment (m-kharbat) [inline app/views/console.html]: yes it's for the width

**Comment by chaudum** [comment]
all fixes also need to be backported to 1.0, since 1.1 has not been released (together with crate) yet.

**Pull request merged**